e2e/qr-code-reader.e2e.ts

```diff
@@ -25,7 +27,9 @@ const testQRCode = async ({
   // Wait for video
   await page.waitForTimeout(2000);
 
-  await expect(page).toHaveScreenshot();
+  screenshotName !== undefined && screenshotName !== ""
```

Comment:
Would the string ever be empty?

Reply:
No but we cannot enforce it TypeScript wise.

Reply:
Does TypeScript complain if you remove the check for `""`?

Reply:
I mean Playwright fails if we path `""` as screenshot name.

Reply:
But why would someone pass `""` as a screenshot name?
Aren't there lots of things you could pass that it would fail on?
Do you want to check all of them?
Like `"/"` or `"."` probably also fail, right?

Reply:
Removed the empty check